pyfarm/agent/service.py

```diff
@@ -77,6 +79,8 @@ class Agent(object):
     itself with the master, starting the periodic task manager,
     and handling shutdown conditions.
     """
+    reactor = reactor
```

Comment:
It's not anymore, removing.  Anytime you see something like this it's usually the reactor can be replaced in testing.  In the case of this change I have not had a reason to use it for testing so far...